pkg/cmdlets/commit/commit.go

```diff
@@ -105,6 +105,13 @@ func FormatChanges(commit *mono_models.Commit) ([]string, []*requirementChange)
 			versionConstraints = ""
 		}
 
+		// This is a temporary fix until we start getting history in the form of build expressions
```

Comment:
Is there a ticket filed for this? We should probably reference here so we don't lose track of this comment.